docs/process/issue-triage.md

```diff
@@ -1,5 +1,113 @@
 # Issue Triage
 
+> Triage (/ˈtriːɑːʒ/ or /triːˈɑːʒ/) is the process of determining the priority
+> of patients' treatments based on the severity of their condition. This
+> rations patient treatment efficiently when resources are insufficient for all
+> to be treated immediately.
+>
+> *From Wikipedia*
+
+The above describes medical triage but it is clear that it also applies to our
+situation. Triage is a process of sifting through all the things that we could
+work on to select the few things that we will work on. In order to maximize the
+impact we have for the people that use GitHub Desktop, things that will get top
+priority are items that are well-described, clearly presented and have obvious
+benefit.
+
+Additionally, we want to encourage helpful feedback and meaningful
+participation. In order to do this, we will have to be clear about what we need
+from people so that we can deliver what they need. This also means that we will
+have to be very clear and decisive when we are not getting the information or
+cooperation we need so that we can move on. Just like in an emergency room, if
+it is a choice between spending several hours to have a 10% chance of saving
+one person or spending several hours definitely saving multiple people, the
+choice is clear.
+
+## Goals
+
+* Communicate clearly and effectively
+    * What the maintainers will work on
+    * What pull requests will be reviewed for acceptance
+    * What pull requests *will not* be reviewed for acceptance
+* Outline exactly what is expected for an issue to meet the "triage bar" so
+  that issues that don't meet the bar can be closed
+* Reduce the amount of time and back-and-forth needed to take an issue from
+  being first-opened to `triaged` or closed
+* Accept input from the community that helps us deliver meaningful results back
+  to the Atom community
+
+## The Issues List Is Our Backlog
+
+The GitHub Desktop issues list is what the maintainers team uses to guide our
+work. In order for our work to be focused and efficient, our issues list must
+be clean and well-organized. Accepting input from the community is a
+significant benefit *when it does not distract us from making things better*.
+
+* Untriaged issues are tasks that are being evaluated to determine if they meet
+  the triage bar
+* Open triaged issues are tasks that the maintainers have agreed to work on
+* Closed issues are things that either didn't meet the triage bar or are tasks
+  that the maintainers will not be taking on
+
+## The Triage Bar
```

Comment:
This doesn't provide enough guidance for issues that aren't bugs... 

For instance, if the user is asking for a new feature or an enhancement to an old one, the triage bar as defined may not be met.

Reply:
I disagree.

1. The build number that the person is using when they request a new feature or enhancement is important because often the feature or enhancement is already there and they don't know how to access it. Or it gets added separately from the issue and we need to know whether the submitter needs guidance or the issue can simply be closed.
1. OS information is similar to :point_up:
1. Reproduction steps are important to ensure that we actually implement what they want rather than what it sounds like they want

Reply:
@lee-dohm  I guess my issue is with the use of the word _problem_. I automatically jumped to the conclusion that the Triage Bar is only about problems. But I can see how you can interpret it that way.

Reply:
All good feedback, but this is essentially a c&p job from the existing document in the issue triage repo. For this PR I don't intend to revisit how we actually triage issues.